### PR TITLE
Update the changelog for Jetpack 13.4

### PR DESCRIPTION
**Pull request by ice9js**
## Proposed changes:

Updates the changelog entries for the upcoming Jetpack 13.4 release.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

N/A

## Testing instructions:

* Proofread changes. In particular "enhancements", "compatibility improvements" and "bug fixes".

### CODE REVIEW

File: projects/plugins/jetpack/CHANGELOG.md [hunk 1]
```diff
@@ -4,182 +4,131 @@
 
 ## 13.4-beta - 2024-05-06
 ### Enhancements
-- Paid Content Block: Swap out the login flow. [#37178]
-- Subscriptions: Manage subscribers in Jetpack Cloud instead of WP.com. [#37155]
-- Subscription widget: Remove "follow" term from confirmation message [#37135]
+- Jetpack AI: Add an AI Featured Image Generator. [#36709] [#36806] [#36812] [#36822] [#36831] [#36863] [#36865] [#36868] [#36899] [#36926] [#36963] [#36965] [#37086] [#37166] [#37209] [#37219]
+- Newsletters: Manage subscribers in Jetpack Cloud. [#37155]
+- Newsletters: Add a "Reply-to" setting for newsletter emails. [#37011]
+- Newsletters: Add filters for controlling the Subscribe Modal load time and the edit capability for the Newsletter Block Neta. [#36909] [#36374]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]: I suspect this should be "Meta"?

```suggestion
- Newsletters: Add filters for controlling the Subscribe Modal load time and the edit capability for the Newsletter Block Meta. [#36909] [#36374]
```

Comment (ice9js) [reply projects/plugins/jetpack/CHANGELOG.md]: Yes 😅

File: projects/plugins/jetpack/CHANGELOG.md [hunk 2]
```diff
@@ -4,182 +4,131 @@
...
-### Enhancements
-- Subscription: Add a filter to Allow Newsletter Block Meta Capability to be Changed. [#36909]
-
-### Other changes <!-- Non-user-facing changes go here. This section will not be copied to readme.txt. -->
-- Add scheduled updates sync option. [#36877]
-- Add in_site_migration_flow to the list of available site options to update and retrieve. This will be used as part of the Site Migration on-boarding flow. [#36974]
-- Adds the ability to filter users in the WPCOM_JSON_API_List_Users_Endpoint endpoint by capability. [#36893]
-- Add the userConnectionUrl to initial state to use in Social sidebar. [#36976]
+- AI Assistant: Add title optimization as beta [#37001]
+- AI Assistant: Add title optimization jetpack sidebar entry. [#37002]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]: We probably don't care about most of these entries, as they're not user-facing. To be honest, I'm not even sure what most of them mean. But probably should capitalize "Jetpack"?

```suggestion
- AI Assistant: Add title optimization Jetpack sidebar entry. [#37002]
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 3]
```diff
@@ -4,182 +4,131 @@
...
 - Blocks: Move localizable strings out of the metadata attributes field. [#36724]
+- Business Hours Block: Refactor Edit component and fix time formatting in preview mode. [#36785] [#36786]
+- CLI: Make `wp jetpack publicize` work again. [#36988]
 - Carousel: Add extra isset check to avoid warnings. [#36681]
-- Create /wpcom/v2/admin-color endpoint that returns admin color scheme. [#36749]
+- Carousel: Ensure that disabling the feature on WordPress.com Simple brings back Core's Lightbox back. [#36914]
+- CloudFlare Analytics: deprecate feature and move to mu-wpcom. [#37061]
+- Contact Form: Removing code and renaming relevant references to ensure we use the package version of the contact form. [#37157]
+- Cookie Consent: Localize default text. [#37012]
+- Donations Block: Fix non localized strings. [#36951]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- Donations Block: Fix non-localized strings. [#36951]
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 4]
```diff
@@ -4,182 +4,131 @@
...
+- Donations Block: Show Stripe nudge if not connected. [#36841]
+- Enhanced Distribution: Remove deprecated module file. [#37032]
 - Eventbrite Block: Refactor Edit component to function. [#36739]
-- Fix translation issues for default attributes. [#36695]
-- Jetpack AI Assistant: Remove old deadcode from image modal. [#36683]
+- Featured Image: Add 'Carrousel' in 'Featured Image'. [#36834]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- Featured Image: Add 'Carousel' to 'Featured Image'. [#36834]
```

**Review by tbradsha — DISMISSED** [review]

File: projects/plugins/jetpack/CHANGELOG.md [hunk 5]
```diff
@@ -4,182 +4,131 @@
...
 ## 13.4-beta - 2024-05-06
 ### Enhancements
-- Paid Content Block: Swap out the login flow. [#37178]
-- Subscriptions: Manage subscribers in Jetpack Cloud instead of WP.com. [#37155]
-- Subscription widget: Remove "follow" term from confirmation message [#37135]
+- Jetpack AI: Add an AI Featured Image Generator. [#36709] [#36806] [#36812] [#36822] [#36831] [#36863] [#36865] [#36868] [#36899] [#36926] [#36963] [#36965] [#37086] [#37166] [#37209] [#37219]
+- Newsletters: Manage subscribers in Jetpack Cloud. [#37155]
+- Newsletters: Add a "Reply-to" setting for newsletter emails. [#37011]
+- Newsletters: Add filters for controlling the Subscribe Modal load time and the edit capability for the Newsletter Block Meta. [#36909] [#36374]
+- Paid Content Block: Login flow is matching the Subscribe Block. [#37178]
```
Comment (anomiex) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- Paid Content Block: Use a login flow matching the Subscribe Block. [#37178]
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 6]
```diff
@@ -4,182 +4,131 @@
...
 - WordAds: Ensure that ads.txt works on subdirectory websites. [#36887]
 
 ### Improved compatibility
-- Theme tools: Devicepx functionality is deprecated. [#37116]
-- Theme tools: Site-logo functionality is deprecated in favour of core supported custom-logo. [#37109]
+- General: WordPress 6.4 is now the minimum required version. [#37047] [#37049]
+- Theme Tools: Site-logo and devicepx are deprecated. Use WP Custom Logo instead. [#37116] [#37109]
```
Comment (anomiex) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- Theme Tools: Site-logo and devicepx are deprecated. Use WordPress Core's custom-logo or logo block instead. [#37116] [#37109]
```

**Review by anomiex — DISMISSED** [review]
Top part looks ok, but left a few suggestions inline.

**Review by anomiex — APPROVED** [review]

**Pull request merged**